src/components/m-button/_template.js

```diff
@@ -33,14 +37,19 @@ export default function ({
     genericIcon = raw(`<axa-icon icon="${icon}" classes="m-button__icon"></axa-icon>`);
   }
 
-  if (tag.toLowerCase() === 'a') {
+  if (tag.toLowerCase() === 'a' && disabled) {
+    return html`<a href="${href}" target="${target}" class="${buttonClasses}" ${ARIA_DISABLED} tabindex="-1">
```

Comment:
Can you use inline check for the disabled param like you did it for the button below?
We could avoid duplicate condition like this.
${disabled ? `${ARIA_DISABLED} tabindex="-1"` : ''}

Reply:
I tried this before, but it didn't work so I switched to this method.

It gives me the error in the console:
`index.js:2780 DOMException: Failed to execute 'setAttribute' on 'Element': 'aria-disabled tabindex="-1"' is not a valid attribute name.
    at nanohtmlSetAttribute`